loader/loader.c

```diff
@@ -3144,7 +3208,7 @@ static VkResult read_data_files_in_search_paths(const struct loader_instance *in
         if (NULL != tmp_search_path) {
             strncpy(tmp_search_path, search_path, search_path_size);
             tmp_search_path[search_path_size] = '\0';
-            if (data_file_type == LOADER_DATA_FILE_MANIFEST_ICD) {
+            if (is_icd) {
```

Comment:
Why replace the json_type with an explicit bool here? (and the few other places `is_icd` is used.